### DISCUSSION TITLE
Duolingo

### DISCUSSION THREAD

**Discussion by janTatesa** [catppuccin/catppuccin#2219] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the suggested port be themed?

Can you please make a stylus style for duolingo? Thanks.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by RichardKanshen** [comment]
I think a duolingo port is well needed as a popular site.

**Comment by janTatesa** [comment]
Started working on it!

**Reply by krymancer** [reply]
I'm glad to help if you need

**Comment by krymancer** [comment]
Since 4 days passed and @TadoTheMiner  didn't said anything I started to make the userstyles. I don't know less so is kind half done for now

Still need a lot of work but I guess is a good start, mocha is the best and frappe needs some love. 

[Here is the draft of the PR](https://github.com/catppuccin/userstyles/pull/1021)
[Here is the repo if anyone wanna help](https://github.com/Krymancer/userstyles/tree/feat/duolingo)

**Reply by janTatesa** [reply]
Oh okay, will help today
